lib/cdn-site-hosting/cdn-site-hosting-with-dns-construct.ts

```diff
@@ -58,9 +58,7 @@ export class CdnSiteHostingWithDnsConstruct extends cdk.Construct {
     new route53.ARecord(this, `${siteDomain}-SiteAliasRecord`, {
       recordName: siteDomain,
       target: route53.RecordTarget.fromAlias(
-        new targets.CloudFrontTarget(
-          this.cdnSiteHosting.cloudfrontWebDistribution
-        )
+        new targets.CloudFrontTarget(this.cdnSiteHosting.cloudfrontDistribution)
```

Comment:
IIRC, this will be used for non-production stacks, so we could use `cdk destroy` then `cdk deploy` without concern.